packages/react/src/components/form/Checkbox/Checkbox.module.css

```diff
@@ -116,19 +133,19 @@
     color: var(--fds-semantic-text-action-hover);
   }
 
-  .container:not(.readonly, .disabled) > .input:hover:not(:checked) + .label::before {
+  .container:not(.readonly, .disabled) > .input:hover:not(:checked) {
     --fds-checkbox-border-color: var(--fds-semantic-border-input-hover);
 
     box-shadow: var(--fds-checkbox-border__hover), inset 0 0 0 2px var(--fds-checkbox-border-color);
   }
 
-  .container:not(.readonly, .disabled) > .input:hover:checked + .label::before {
+  .container:not(.readonly, .disabled) > .input:hover:checked {
     --fds-checkbox-border-color: var(--fds-semantic-border-input-hover);
 
     box-shadow: var(--fds-checkbox-border__hover), inset 0 0 0 2px var(--fds-checkbox-border-color);
   }
 
-  .container:not(.readonly, .disabled) > .input:hover:checked:focus-visible + .label::before {
+  .container:not(.readonly, .disabled) > .input:hover:checked:focus-visible::before {
```

Comment:
This looks abit weird 😅 

![image](https://github.com/digdir/designsystemet/assets/1070981/fb0f4d2c-546c-4dd0-8e29-bed77dd7ca7f)

Looks like missed removing a `::before` when changing to styling on `input` element.

I think this suggestion should fix it 🤔 
```suggestion
  .container:not(.readonly, .disabled) > .input:hover:checked:focus-visible {
```